docs/installingNative.md

```diff
@@ -71,22 +71,11 @@ Now you can install the other dependencies.
 
```

Comment:
Should probably update the yarn through npm section on ubuntu as well to mirror `corepack`.

Reply:
Done in 321d07d99fa8c01b79a45732c944bff110987f84